keanu-python/docs/distributions.rst

```diff
@@ -0,0 +1,28 @@
+Distributions
+=============
+This lists all of the distributons currently available in Keanu Python.
+
+* :meth:`keanu.vertex.Bernoulli`
+* :meth:`keanu.vertex.Beta`
+* :meth:`keanu.vertex.Cauchy`
+* :meth:`keanu.vertex.ChiSquared`
+* :meth:`keanu.vertex.Dirichlet`
+* :meth:`keanu.vertex.Exponential`
+* :meth:`keanu.vertex.Gamma`
+* :meth:`keanu.vertex.Gaussian`
+* :meth:`keanu.vertex.HalfCauchy`
+* :meth:`keanu.vertex.HalfGaussian`
+* :meth:`keanu.vertex.InverseGamma`
+* :meth:`keanu.vertex.Laplace`
+* :meth:`keanu.vertex.LogNormal`
+* :meth:`keanu.vertex.Logistic`
+* :meth:`keanu.vertex.MultivariateGaussian`
+* :meth:`keanu.vertex.Pareto`
+* :meth:`keanu.vertex.SmoothUniform`
+* :meth:`keanu.vertex.StudentT`
+* :meth:`keanu.vertex.Triangular`
+* :meth:`keanu.vertex.Uniform`
+* :meth:`keanu.vertex.Binomial`
+* :meth:`keanu.vertex.Multinomial`
+* :meth:`keanu.vertex.Poisson`
+* :meth:`keanu.vertex.UniformInt`
```

Comment:
Perhaps we need to split this out in to continuous (Double) vs discrete (Integer)?

Reply:
Will leave it as it is for this release. Will add a note for when we automatically generate to split them up.